examples/evpn/cni-underlay/prepare.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+set -x
+CURRENT_PATH=$(dirname "$0")
+
+source "${CURRENT_PATH}/../../common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../common.sh was not specified as input (see shellcheck -x).

(SC1091)